mini_project_1.ipynb

```diff
@@ -0,0 +1,175 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'the'"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from pattern.web import *\n",
+    "import string\n",
+    "\n",
+    "christianity=URL(\"http://www.gutenberg.org/cache/epub/8294/pg8294.txt\").download()\n",
+    "\n",
+    "\n",
+    "\n",
+    "                \n",
+    "def process_file(filename):\n",
```

Comment:
It would be good to add a comment describing what this function does at the top of the function, such as:
```
def process_file(file):
    '''
    This function does something blah blah
    '''
    code here...
```